packages/eas-cli/src/project/publish.ts

```diff
@@ -196,6 +210,43 @@ export function loadMetadata(distRoot: string): Metadata {
   return metadata;
 }
 
+/** Try to load the asset map for logging the names of assets published */
+export async function loadAssetMapAsync(distRoot: string): Promise<AssetMap | null> {
+  const assetMapPath = path.join(distRoot, 'assetmap.json');
```

Comment:
when is this file created? I notice that when I run `eas update` it isn't created. 

I also don't see a `manifest.json` as mentioned in this PR's summary?

Reply:
assetmap is generated with `npx expo export --dump-assetmap`